src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -74,7 +75,7 @@ private enum Flags : ushort
 
 #nullable enable
         private SynthesizedBackingFieldSymbol? _lazyDeclaredBackingField;
-        private SynthesizedBackingFieldSymbol? _lazyMergedBackingField;
+        private StrongBox<SynthesizedBackingFieldSymbol?>? _lazyMergedBackingField;
```

Comment:
I didn't understand what is accomplished by using a StrongBox here, could you please explain?

Reply:
> I didn't understand what is accomplished by using a StrongBox here, could you please explain?

`null` is a valid value for the backing field. Therefore, `null` couldn't be used as a sentinel value (a value representing a not initialized state). But it was used this way in ```BackingField.get```. Potentially causing us to get into the `if` over and over again. Also there was no guarantee that we wouldn't get into the `if` for a partial case. StrongBox allows us to know for sure whether the value is initialized, `null` - not initialized, `not null` initialized.

Reply:
Thank you